emulators/qemu/Portfile

```diff
@@ -52,23 +52,15 @@ patchfiles-append       patch-qemu-configure.diff
 # "error: unknown type name 'fpunchhole_t'" pre-10.12.4
 patchfiles-append       patch-qemu-punchhole.diff
 
-# "cp: illegal option -- a"
-patchfiles-append       patch-qemu-scripts-entitlements.diff
-
 # IPPROTO workarounds
 patchfiles-append       patch-qemu-net-colo.diff
 
 # MADV_DONTNEED workaround
-patchfiles-append       patch-qemu-include-qemu-osdep.diff
+patchfiles-append       patch-qemu-include-qemu-madvise_h.diff
```

Comment:
The rename seems reasonable as the header file was renamed as well. But the old patch file needs to be deleted as well.

Reply:
Hi @raimue,

I have added a dbus variant, a virtfs variant (for 9pfs), and removed the unneeded patch file.  

Thanks,

-ranga

Reply:
Thank you! This is getting closer to what we need for the qemu 7.0 port. Please see the other inline comments regarding details for these new features.